payments-core/src/main/java/com/stripe/android/payments/core/injection/AuthenticationComponent.kt

```diff
@@ -12,7 +23,57 @@ import javax.inject.Singleton
  * into it.
  */
 @Singleton
-@Component(modules = [AuthenticationModule::class])
+@Component(
+    modules = [
+        AuthenticationModule::class,
+        Stripe3DSAuthenticatorModule::class
+    ]
+)
 internal interface AuthenticationComponent {
     val registry: DefaultIntentAuthenticatorRegistry
+
+    @Component.Builder
+    interface Builder {
+        @BindsInstance
+        fun context(context: Context): Builder
```

Comment:
Maybe out of scope for this PR, but can we depend on `Application` instead of `Context`? That's what we really want I think?

Reply:
This context is current passed from `StripePaymentController`, which gets it from our public interfaces in `Stripe` and `PaymentSheet`. We can change this to Application once the public APIs take it as a parameter